src/output.c

```diff
@@ -536,46 +498,6 @@ void OutputRegisterFiledataModule(LoggerId id, const char *name,
     FatalError("Fatal error encountered. Exiting...");
 }
 
-/**
- * \brief Register a file data output sub-module.
- *
- * This function will register an output module so it can be
- * configured with the configuration file.
- *
- * \retval Returns 0 on success, -1 on failure.
- */
-void OutputRegisterFiledataSubModule(LoggerId id, const char *parent_name,
```

Comment:
I suppose some plugin author could potentially be using any output registration functions.

Reply:
What would be a file data submodule ?

Reply:
Oh, I see we weren't using this at all.  I thought we were just removing the public exposure.